src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/CcdEventController.kt

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.controllers
+
+import org.springframework.http.MediaType.APPLICATION_JSON_VALUE
+import org.springframework.http.ResponseEntity
+import org.springframework.http.ResponseEntity.ok
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest
+import uk.gov.hmcts.reform.ccd.client.model.CallbackResponse
+
+@RestController
+class CcdEventController {
+    @PostMapping(
+        value = ["/callback/{type}"],
```

Comment:
more clear `path` _could_ be used

Reply:
What exactly is unclear about it ?
I  initially had /callbacks/ccd/{type} in the spike I think tim and nitin like it this way instead.
Its really self explanatory though ... callback and type of of the callback ... maybe the path Parameter could be named a bit better ?
`/callback/{callbackType}` ?

Reply:
its fine imo, could do
`/ccd/callback/type`

Reply:
Sure, since its only ever going to be CCD calling back it really does not matter.

Reply:
I meant that there's an alias for `value` in this annotation called `path` :)

Reply:
ah ... thats what you mean  ... sure Ill do that.